.changeset/hungry-plants-behave.md

```diff
@@ -0,0 +1,30 @@
+---
+"@salt-ds/lab": patch
+---
+
+Updated Switch's styling
+Refactored Switch and updated its change handler.
```

Comment:
should we say ControlLabel removed?

Reply:
Ah I forgot I did that, I was debating whether to keep it or delete it